TG-23 Make a GET HTTP Request for getting a list of games

jpadgett7 opened the pull request:
Just want to know if I've got the container/component architecture established correctly

jpadgett7 committed:
Made a GET HTTP request to get list of games from the server

renodubois commented on containers/create_request/create_request_container.js:
```diff
@@ -0,0 +1,31 @@
+
+  render () {
+    return (
+      <CreateRequestComponent games={this.gamesList} />
```
This should be `this.state.gamesList`

renodubois commented on components/create_request/create_request_component.js:
```diff
@@ -36,35 +38,42 @@ export default class CreateRequestComponent extends React.Component {
+            <Item floatingLabel>
+              <Label>Post Title</Label>
+              <TextBox padder
+                     onChangeText={(input) => this.setState({postTitle: input})} />
```
You can make a generic `handleInputChange` for all of your form events, the React docs go over this pretty well: https://reactjs.org/docs/forms.html#handling-multiple-inputs

renodubois reviewed (dismissed):
Haven't ran it, but assuming it works (as much as it can w/o an actual URL).

Here are a few comments, let me know if you have questions on them.

jpadgett7 committed:
Added PUSH request to CreateRequestComponent
Created generic function to handle form input changes
Created function to handle Submit button press
some of this might not work because Expo /shrug

jpadgett7 committed:
Updated HTTP requests to connect to tangle-server

jpadgett7 committed:
added package-lock.json

jpadgett7 committed:
added axios to package.json

jpadgett7 committed:
Actually fixed package.json this time

jpadgett7 committed:
Adjusted the GET request in the CreateRequest container

jpadgett7 committed:
Fixed the serverAddress not being used correctly container

jpadgett7 committed:
fixed promise hell in container

jpadgett7 committed:
Fine-tuned the HTTP requests

jpadgett7 committed:
Minor fix to POST request

jpadgett7 committed:
Another minor fix

jpadgett7 committed:
Refactored TextBox back to Native Base's Input component

jpadgett7 committed:
Removed handleInputChange() as it was causing errors

jpadgett7 committed:
my naming scheme is trash

jpadgett7 committed:
Fixed the picker

renodubois committed:
Fixes gamePicker items

michael-beaver committed:
Merge branch 'develop' into TG-23

# Conflicts:
#	package-lock.json
#	package.json

jpadgett7 committed:
bug test this thing

jpadgett7 committed:
CreateRequest now navigates to newly created request on submit

michael-beaver reviewed (approved):
Tested, looks functional. We need to fix the issue with the game selection soon though.
